src/main/kotlin/it/unibo/protelis/web/execution/CoroutineProtelisEngine.kt

```diff
@@ -0,0 +1,30 @@
+package it.unibo.protelis.web.execution
+
+import java.util.concurrent.CompletableFuture
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.future.asCompletableFuture
+import kotlinx.coroutines.launch
+
+abstract class CoroutineProtelisEngine : ProtelisEngine {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [UndocumentedPublicClass - [CoroutineProtelisEngine] at /src/src/main/kotlin/it/unibo/protelis/web/execution/CoroutineProtelisEngine.kt:8:1](https://app.codacy.com/manual/NiccoMlt/Protelis-Web/pullRequest?prid=4969516)